### ISSUE TITLE
Problem with WiFi hotspot on Mate 10 Pro

### ISSUE THREAD

**Issue by Vittfarne** [phhusson/treble_experimentations#108]
Activating WiFi hotspot on my mate 10 pro results in the settings app displaying "Error" under the option to enable/disable and it's not possible to connect to the WiFi hotspot, the network is broadcasting but connecting is impossible.

I've attached the logcat and tapped the screen lot's of times to make it easier to find where the log starts just before turning on the hotspot, the line `07-02 20:59:22.499  9246  9246 I ConnectivityManager: startTethering caller:com.android.settings` is the first line after activating the hotspot.

[logcat3.txt](https://github.com/phhusson/treble_experimentations/files/2156532/logcat3.txt)

Phone details:

| Option    | Value           | 
| ------------- |:-------------:| 
| Model       | BLA-L29 | 
| Image used/Release      | v21 A-only ARM64 devices with GApps and SU |  

Let me know if there is anything I can do to help solve the issue!

**Comment by phhusson** [comment]
logs got eaten by chatty
run adb logcat -P ''; adb logcat -c
then redo the logs

**Comment by Vittfarne** [comment]
Thanks!

Windows did not like the single quotes at all, ran with double quotes instead and reran the logcat:
[logcat4.txt](https://github.com/phhusson/treble_experimentations/files/2156613/logcat4.txt)

**Comment by phhusson** [comment]
Ooops
That's actually a bug I fixed in november in 8.0 that I forgot to backport to 8.1 ( https://github.com/phhusson/platform_system_netd/commit/d0345c910faf71b26ca77955dc8c6127ff1f1586 )

Thanks for the report!

**Comment by phhusson** [comment]
Should be fixed with
https://github.com/phhusson/platform_system_netd/commit/a00416e37bf6375fb5f43585a9d514a6b63b394e

**Comment by phhusson** [comment]
Hum, Android has support for 5GHz hotspot, but you have to switch it by hand in the hotspot setting
It won't work in my GSI (yet, that's a wip)

Totally unrelated to my GSI, would you be interested in a system aggregating bandwidth and/or improve stability by connecting two phones' internet connection together?

**Comment by Vittfarne** [comment]
Yeah, seeing that I even had an option for frequency in this ROM is what made me check, can't find any option for it on my S8 at all and in iOS, haha, not happening ever I would think.

I've been using [Speedify](https://speedify.com/) to bond two connections, but for connections above ~40Mbit/s, which pretty much means any connection I use here in Sweden, I get the same speed as one of the connections alone, or, slower (pretty much always happens for upload, just tried now, without I get ~200Mbit/s + ~20Mbit/s but bonded I get ~80Mbit/s which means I lose more than 50% in speed). Works great for redundancy when moving between cell towers quick (like on a train) and using multiple carriers, or for bonding slow connections (<15Mbit/s). Got anything you think would work better or works in a different way I would be happy to test it! 😄
